spec/migrations/20200512201614_update_chargeback_startpage_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_migration
+
+describe UpdateChargebackStartpage do
+  let(:user_stub) { migration_stub :User }
+
+  migration_context :up do
+    describe 'starting page replace' do
+      it 'replaces user starting page if chargeback/explorer' do
+        # feature_stub.create!(:identifier => 'chargeback')
```

Comment:
Don't think we need a comment as it can just be removed.

Reply:
@Fryguy removed the leftover commented line